.changeset/six-pandas-fix.md

```diff
@@ -0,0 +1,8 @@
+---
+'@shopify/ui-extensions': minor
+'@shopify/ui-extensions-react': minor
+---
+
+The `purchase.checkout.shipping-option-list.render-before` and `purchase.checkout.shipping-option-list.render-after` extension targets now include a `target` to represents the delivery group it is rendering for.
```

Comment:
Should we mention the duplication behaviour change, or is this changeset just for the API changes we're making?